2019/10/24 10:26:17 [error] [exception.ParseError] ParseError: syntax error, unexpected '$this' (T_VARIABLE), expecting function (T_FUNCTION) or const (T_CONST) in C:\xampp\htdocs\yii\protected\controllers\PostController.php:5
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CWebApplication->createController('')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#2 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#3 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#4 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:27:13 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:27:16 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:27:38 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:27:49 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/post/index
HTTP_REFERER=http://localhost:81/yii/index.php/site/page?view=about
---
2019/10/24 10:28:11 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/post/index
---
2019/10/24 10:28:15 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:28:40 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:29:16 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:29:28 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/post/index
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/24 10:29:29 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/post/index
HTTP_REFERER=http://localhost:81/yii/index.php/post/index
---
2019/10/24 10:29:30 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/post/index
HTTP_REFERER=http://localhost:81/yii/index.php/post/index
---
2019/10/24 10:30:22 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/post/index
---
2019/10/24 10:30:27 [error] [exception.CException] CException: PostController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/index')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/post/index
---
2019/10/24 10:36:36 [error] [exception.CException] CException: Property "PostController.breadcrumbs" is not defined. in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\protected\views\layouts\main.php(41): CComponent->__get('breadcrumbs')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(787): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\protected\controllers\PostController.php(7): CController->render('login')
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): PostController->actionIndex()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#11 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#13 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:55:22 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "login". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:55:36 [error] [exception.CException] CException: LoginController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\LoginController.php(9): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): LoginController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/24 10:56:31 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "post/login". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('post/login')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/post/login
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/24 10:58:23 [error] [php] Undefined index: username (C:\xampp\htdocs\yii\protected\controllers\HomeController.php:13)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): HomeController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): HomeController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): HomeController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\index.php(11): CWebApplication->run()
REQUEST_URI=/yii/index.php/home/login
2019/10/24 10:59:46 [error] [php] Undefined index: username (C:\xampp\htdocs\yii\protected\controllers\HomeController.php:13)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): HomeController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): HomeController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): HomeController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\index.php(11): CWebApplication->run()
REQUEST_URI=/yii/index.php/home/login
2019/10/24 11:15:19 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dashboard". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dashboard/
---
2019/10/24 11:15:22 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dashboard". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dashboard
---
2019/10/24 11:15:26 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dashboard/index". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard/index')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dashboard/index
---
2019/10/24 11:15:43 [error] [php] Undefined index: username (C:\xampp\htdocs\yii\protected\controllers\HomeController.php:13)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): HomeController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): HomeController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): HomeController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\index.php(11): CWebApplication->run()
REQUEST_URI=/yii/index.php/home/login
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user2-160x160.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user2-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user2-160x160.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user8-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user8-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user8-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user1-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user1-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user1-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/AdminLTELogo.png". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/AdminL...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/AdminLTELogo.png
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user5-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user5-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user5-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user6-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user6-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user6-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user7-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user7-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user7-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:16:08 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user3-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user3-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user3-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user1-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user1-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user1-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user8-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user8-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user8-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user3-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user3-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user3-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/AdminLTELogo.png". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/AdminL...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/AdminLTELogo.png
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user6-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user6-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user6-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user7-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user7-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user7-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:24 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user5-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user5-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user5-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:54 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user1-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user1-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user1-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:54 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user6-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user6-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user6-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:54 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user8-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user8-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user8-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:55 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user5-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user5-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user5-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:55 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user3-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user3-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user3-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:17:55 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user7-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user7-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user7-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:18:41 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user1-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user1-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user1-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:18:41 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user3-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user3-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user3-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:18:41 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user7-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user7-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user7-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:18:41 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user6-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user6-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user6-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:18:41 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user8-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user8-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user8-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:18:41 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "dist/img/user5-128x128.jpg". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dist/img/user5-...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/dist/img/user5-128x128.jpg
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/24 11:20:51 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "index3.html". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('index3.html')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/index3.html
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/25 09:58:56 [error] [exception.Error] Error: Access to undeclared static property: Yii::$app in C:\xampp\htdocs\yii\protected\controllers\HomeController.php:17
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionLogout()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('logout')
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/logout')
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#7 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#8 {main}
REQUEST_URI=/yii/index.php/home/logout
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/25 10:06:34 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "dashboard". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('dashboard')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('dashboard')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/dashboard')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/dashboard
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/25 10:06:52 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "dashboard". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('dashboard')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('dashboard')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/dashboard')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/dashboard
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/25 10:06:53 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "dashboard". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('dashboard')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('dashboard')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/dashboard')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/dashboard
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/25 10:06:55 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "dashboard". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('dashboard')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('dashboard')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/dashboard')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/dashboard
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/25 10:06:55 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "dashboard". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('dashboard')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('dashboard')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/dashboard')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/dashboard
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/25 10:06:55 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "dashboard". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('dashboard')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('dashboard')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/dashboard')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/dashboard
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/25 10:06:56 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "dashboard". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('dashboard')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('dashboard')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/dashboard')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/dashboard
HTTP_REFERER=http://localhost:81/yii/
---
2019/10/25 10:12:37 [error] [exception.CException] CException: HomeController and its behaviors do not have a method or closure named "goHome". in C:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(44): CComponent->__call('goHome', Array)
#1 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionLogin()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/login')
#7 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#8 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#9 {main}
REQUEST_URI=/yii/index.php/home/login
HTTP_REFERER=http://localhost:81/yii/index.php/home/Index
---
2019/10/25 10:21:53 [error] [exception.CException] CException: CHttpSession and its behaviors do not have a method or closure named "has". in C:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\yii\protected\controllers\DashboardController.php(7): CComponent->__call('has', Array)
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(359): DashboardController->__construct('dashboard', NULL)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CWebApplication->createController('')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard')
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#6 {main}
REQUEST_URI=/yii/index.php/dashboard
---
2019/10/25 10:21:55 [error] [exception.CException] CException: CHttpSession and its behaviors do not have a method or closure named "has". in C:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 C:\xampp\htdocs\yii\protected\controllers\DashboardController.php(7): CComponent->__call('has', Array)
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(359): DashboardController->__construct('dashboard', NULL)
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(276): CWebApplication->createController('')
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard')
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#6 {main}
REQUEST_URI=/yii/index.php/dashboard
---
2019/10/25 10:22:52 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/home
---
2019/10/25 10:23:12 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:23:16 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:24:15 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:24:16 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:24:29 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/home
---
2019/10/25 10:27:01 [error] [exception.CException] CException: HomeController cannot find the requested view "mainlogin". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('mainlogin', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('mainlogin')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/home
---
2019/10/25 10:27:02 [error] [exception.CException] CException: HomeController cannot find the requested view "mainlogin". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('mainlogin', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('mainlogin')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/home
---
2019/10/25 10:27:08 [error] [exception.CException] CException: HomeController cannot find the requested view "mainlogin". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('mainlogin', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('mainlogin')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:27:18 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:27:19 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:33:19 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:33:34 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:35:05 [error] [exception.CException] CException: HomeController cannot find the requested view "index". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('index', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('index')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:35:14 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:35:19 [error] [exception.CException] CException: HomeController cannot find the requested view "login". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('login', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\HomeController.php(17): CController->render('login')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HomeController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/
---
2019/10/25 10:36:30 [error] [exception.CException] CException: DashboardController cannot find the requested view "home". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('home', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\DashboardController.php(15): CController->render('home')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DashboardController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/dashboard
HTTP_REFERER=http://localhost:81/yii/index.php/home
---
2019/10/25 10:37:12 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "index". in C:\xampp\htdocs\yii\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('index')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('home/index')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#5 {main}
REQUEST_URI=/yii/index.php/home/index
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
2019/10/25 10:44:22 [error] [php] Illegal offset type in isset or empty (C:\xampp\htdocs\yii\framework\web\CHttpSession.php:575)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): HomeController->runAction()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): HomeController->runActionWithFilters()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): HomeController->run()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#6 C:\xampp\htdocs\yii\index.php(11): CWebApplication->run()
REQUEST_URI=/yii/index.php/home/login
2019/10/25 10:47:13 [error] [exception.CException] CException: DashboardController cannot find the requested view "home". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('home', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\DashboardController.php(15): CController->render('home')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DashboardController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/dashboard
HTTP_REFERER=http://localhost:81/yii/index.php/home/index
---
2019/10/25 10:47:22 [error] [exception.CException] CException: DashboardController cannot find the requested view "home". in C:\xampp\htdocs\yii\framework\web\CController.php:881
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('home', NULL, true)
#1 C:\xampp\htdocs\yii\protected\controllers\DashboardController.php(15): CController->render('home')
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DashboardController->actionIndex()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('dashboard')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#9 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#10 {main}
REQUEST_URI=/yii/index.php/dashboard
---
2019/10/25 10:59:46 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "login". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/login
---
2019/10/25 10:59:51 [error] [php] Undefined index: username (C:\xampp\htdocs\yii\protected\controllers\HomeController.php:22)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): HomeController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): HomeController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): HomeController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#5 C:\xampp\htdocs\yii\index.php(11): CWebApplication->run()
REQUEST_URI=/yii/index.php/home/login
2019/10/25 11:21:39 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "index3.html". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('index3.html')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\index.php(11): CApplication->run()
#3 {main}
REQUEST_URI=/yii/index.php/index3.html
HTTP_REFERER=http://localhost:81/yii/index.php/dashboard
---
